core/templates/pages/release-coordinator-page/beam-jobs-tab/beam-jobs-tab.component.spec.ts

```diff
@@ -220,7 +220,7 @@ describe('Beam Jobs Tab Component', () => {
     expect(await table.getRows()).toHaveSize(2);
 
     component.ngOnDestroy();
-  });
+  }, 10000);
```

Comment:
Which step here in particular is taking a long time? Are these tests failing on CI or your local machine?

Same question for other tests. The concern I have is that these tests aren't failing elsewhere (AFAIK) so I don't understand what is causing this timeout for this PR specifically.

Reply:
The tests will fail randomly on my local machine, and is usually one or two of them that fail at a time on each run. I provided an example screenshot of what one of these errors look like in the proof of changes section in the PR. The reason I changed it was because I unfortunately couldn't get past these errors in order to push my code for the actual issue and create this PR. Modifying the timeout is what allowed me to do so.

I also ran into these timeout errors when running the original/current code repository, so it's unrelated to my changes and could be related to my local machine only. If so, is it possible for this change to omitted without me having to push the code without this timeout modification? Or if there's something else that I need to do to make it pass on my machine?

Reply:
Can you investigate (perhaps by commenting out bits) which line of the test in particular is taking a long time on your machine?

Reply:
Was able to push without it, so these modifications have been reverted to the original code.